src/plugins/console/public/services/history.ts

```diff
@@ -9,6 +9,12 @@
 import { BehaviorSubject } from 'rxjs';
 import { Storage } from './index';
 
+const MAX_NUMBER_OF_HISTORY_ITEMS = 100;
+
+export const errors = {
```

Comment:
nit: when I initially looked at this object I thought it was going to be a map to hold errors. I think the confusion came from the name being lowercase. Had it been `Errors` then it would mimic a class with a static method `isQuotaExceeded`. (consumed as `Errors.isQuotaExceeded(e)`.

I wonder if we need the extra object and if `export const isQuotaExceededError() => {}` wouldn't be enough.